_1327/minutes/models.py

```diff
@@ -114,7 +114,7 @@ def show_publish_button(self):
 		return not self.is_in_creation and self.state == MinutesDocument.UNPUBLISHED
 
 	def publish(self, state_id):
-		if state_id not in (MinutesDocument.PUBLISHED, MinutesDocument.PUBLISHED_STUDENT):
+		if state_id in (MinutesDocument.PUBLISHED, MinutesDocument.PUBLISHED_STUDENT):
```

Comment:
why did this *ever* work? when testing on the current master i had troubles publishing a minutes document because the server always returned `SuspiciousOperation`. it works on production, but this code didn't change since it's introduction.

Reply:
`state_id` is a string and not an integer, this is the reason for the failure.

Furthermore, it might make sense to use rename `state_id` to `next_state_id` or `new_state_id`